### DISCUSSION TITLE
홈 모바일 UI 디자인

### DISCUSSION THREAD

**Discussion by daehwan2** [code-etc/Documents#6] [Ideas]
@yunjin-kim 
디자인에 도움을 드리고 싶어서 모바일 UI 디자인을 한번 해보았습니다. 
![image](https://user-images.githubusercontent.com/53414542/151398565-c72f1ecd-2f45-4997-ba3b-dcaf5ee6d939.png)
<br/>
![image](https://user-images.githubusercontent.com/53414542/151398713-63b8e62d-0879-4eb1-8f76-8651e56e8444.png)

**Comment by yunjin-kim** [comment]
오옹 좋은 것 같습니다 
모바일은 이대로 진행해도 괜찮을 것 같네요!

**Comment by on-Sync** [comment]
굿굿 ^.^
